tensorflow/core/tpu/tpu_initializer_helper.cc

```diff
@@ -51,6 +54,62 @@ bool GetEnvBool(const char* name, bool defval) {
 
 }  // namespace
 
+bool PrintFdMessage(long tgid) {
+  char path[40], line[100];
+  DIR* fd;
+  struct dirent* ent;
+  long deviceId;
+  snprintf(path, 40, "/proc/%ld/fd", tgid); 
+  fd = opendir(path);
+  if(!fd) {
```

Comment:
It looks like there are some minor style issues, e.g. there should be a space after if: `if (`. Rather than review + fix all these by hand, you should look up how to install + run clang-format, which will automatically format everything (or you can possibly do it from your text editor/IDE). Lemme know if you need help with this.